### ISSUE TITLE
document recommended mime-type for usdz

### ISSUE THREAD

**Issue by smalls** [GoogleWebComponents/model-viewer#350]
There may be recommendations for what mime-type USDZ should be served as. Unfortunately, the actual recommendation isn't yet clear.

Once that's clear, we should try to use that mime type in our USDZ samples, and update our documentation with the same recommendation.

Apple's Quick Look gallery uses `model/usd`:

```
> curl -I https://developer.apple.com/arkit/gallery/models/stratocaster/stratocaster.usdz
...
Content-Type: model/usd
...
```

WebKit seems to [recommend](https://webkit.org/blog/8421/viewing-augmented-reality-assets-in-safari-for-ios/) `model/vnd.pixar.usd`:

```
> curl -I https://webkit.org/demos/ar/heart.usdz
...
Content-Type: model/vnd.pixar.usd
...
```

**Comment by jsantell** [comment]
During launch, `model/vnd.usdz+zip` was from the [IANA registration](https://www.iana.org/assignments/media-types/model/vnd.usdz+zip) -- according to that Webkit link, Safari 12.2 and onward must use `model/vnd.usdz+zip`, and previously used `model/vnd.pixar.usd`

**Comment by skabbit** [comment]
Apple's Quick Look gallery still uses model/usd, but WebKit manual suggests to use model/vnd.usdz+zip
And still, no documentation explains the proper way.

**Comment by mds-dwa** [comment]
https://graphics.pixar.com/usd/release/spec_usdz.html says ```subtype name of vnd.usd+zip```, missing the 'z' after usd like the IANA registration uses.

**Comment by elalish** [comment]
@mds-dwa Any idea which is correct?

**Comment by mds-dwa** [comment]
Not really, but I'm assuming "vnd.usdz+zip" since that matches the IANA registration.
